google-cloud-bigtable/test/google/cloud/bigtable/table/read_rows_test.rb

```diff
@@ -434,4 +434,109 @@ def mock.read_rows _args
     _(rows.length).must_equal 1
     _(rows.first).must_equal expected_row
   end
+
+  it "TestReadRows_NoRetry_OutOfOrderError" do
```

Comment:
Describe the functionality rather than use the conformance test name. e.g.

```ruby
it "reports an error if the client detects that the server sent row keys out of order" do
```